server/jetstream_helpers_test.go

```diff
@@ -1307,6 +1322,9 @@ func (c *cluster) waitOnPeerCount(n int) {
 			leader = c.leader()
 		}
 	}
+	antithesis.AssertUnreachable(c.t, "Timeout in cluster::waitOnPeerCount", map[string]any{
```

Comment:
It's a minor nit but `::` isn't Go-standard notation, let's just stick with `.`.